plugins/search/src/components/SearchModal/SearchModal.tsx

```diff
@@ -52,7 +52,7 @@ export interface SearchModalChildrenProps {
 /**
  * @public
  */
-export interface SearchModalProps {
+export interface SearchModalProps extends SearchResultChildrenProvider {
```

Comment:
Here is the only place where I would recommend defining the `resultItems` property.
```suggestion
export interface SearchModalProps extends SearchResultChildrenProvider {
  /**
   * Optional search result item extensions list.
   */
  resultItems?: JSX.Element | ((resultSet: SearchResultSet) => JSX.Element);
```

Reply:
I think I see what you're saying, will take a look